Multi-messenger section (#17)
CAPIBARA3/cosmos-papers: science-case/main.tex

%====================================================
\section{Multi-Messenger Astronomy}
As previously discussed, multi-messenger astronomy is a new way of observing the universe by combining different types of information, these are primarily electromagnetic waves, cosmic rays, neutrinos and gravitational waves. This section covers the fundamental questions about the topic and then proceeds to show how a student led mission can work towards the new way of understanding the cosmos.

> 

Reviewer: - Remove the "as previously discussed," since this is the first section after the introduction.
- instead of "a new way of observing the universe" (sensationalistic), "mma is the term that refers to a recently developed approach to studying the universe by combining different, independent sources of information."
- remove the "primarily", since these 4 are the only ones that are used today I think, double-check that first.
- There is no need to discuss the structure of the section I think.

> 

Reviewer: ```suggestion
Multi-messenger astronomy (MMA) is the term that refers to a recently developed appraoch to studying the universe by combining different, independent sources of information. These sources consist of four fundamental 'messengers': electromagnetic radiation, cosmic rays, neutrinos, and gravitational waves.

```

> First of all, a clarification about these "messengers" is needed: they are different ways of measuring and observing cosmic events. The most historically used is light, which is a spectrum of electromagnetic waves going from radio waves all the way to gamma rays; due to its widely spread range of wavelength, it has to be divided in smaller bands and observed with wavelength specific telescopes. To get a deeper understanding, astronomers commonly combine different bands together, with the clearest example being the production of images through the combination of different photos taken in various bands (multi-band imaging).

Reviewer: - For centuries, the only 'messenger' to observe cosmic events was light. First in the optical range, and later across the electromagnetic spectrum from radio waves to $\gamma$-rays. Due ... imaging).

> First of all, a clarification about these "messengers" is needed: they are different ways of measuring and observing cosmic events. The most historically used is light, which is a spectrum of electromagnetic waves going from radio waves all the way to gamma rays; due to its widely spread range of wavelength, it has to be divided in smaller bands and observed with wavelength specific telescopes. To get a deeper understanding, astronomers commonly combine different bands together, with the clearest example being the production of images through the combination of different photos taken in various bands (multi-band imaging).

Reviewer: ```suggestion
For centuries, the only 'messenger' used to observe cosmic events was light. Initially, limited to the optical range, observations eventually expanded across the entire electromagnetic spectrum, from radio waves to $\gamma$-rays. Because this spectrum is so vast, it must be divided into smaller bands observed with wavelength-specific telescopes. To gain a deeper understand of astrophysical sources, astronomers routinely combine data from these different bands (multi-band imaging).
```

> Recently researchers have switched their focus towards a more complementary approach, in fact, the universe does not only emit light but also provides information through other probes; cosmic rays are one of them, charged particles with relativistic speed coming from our sun primarily but also from extreme astrophysical phenomena happening everywhere in space.

Reviewer: ```suggestion
Modern astrophysics has increasingle embraced the full multi-messenger approach moving beyond just photons \cite{2020_astro_decadal}. The universe provides critical information through other probes that carry unique signatures of extreme astrophysical phenomena. These include
\begin{itemize}
    \item cosmic rays: charged particles travelling at relativistic speeds, originating from our sun and deep-space events \cite{missing}
    \item neutrinos: nearly massless particles that traverse the universe with minimal interaction, offering pristine and early insights into sources like supernovae, active galactic nuclei (AGN) and black holes \cite{missing}
    \item gravitational waves (GWs): ripples in the fabric of space-time generated by the acceleration of massive, super-dense objects (black holes and neutron stars), detectable by highly sensitive interferometers \cite{missing}
\end{itemize}
```

Neutrinos are nearly massless particles which travel through the Universe with minimal interaction, providing unique insights of their sources which are supernovae, black holes, AGN etc. Finally there are gravitational waves: produced by the motion of masses through the fabric of space-time, the ones produced by super-dense objects can be detected by highly-sensitive interferometers.
The majority of these messengers are connected to supernovae explosions, fusion of black holes or neutron stars and overall extreme energy conditions.

> 

Reviewer: ```suggestion
```

> Multi-messenger astronomy was born on 17 August 2017 when a gravitational wave detection was followed by a gamma ray observation \cite{Goldstein2017} (should I explain more in detail what was found and how they detected GW?)

Reviewer: ```suggestion
A paradigm shift in this field occurred on 17 August 2017 (GW170817), when a gravitational wave detection of a neutron star merger was directly followed by the observation of a gamma-ray burst (GRB) \cite{Goldstein2017}. From that moment on, the scientific community's goal has been to synthesize these different types of signals to construct a comprehensive picture of the cosmos. Relying on photons alone is no longer sufficient, observing extreme energy conditions requires the combined "senses" of all available messengers to get the full perspective.
```

> From that moment on, the scientific community's goal was to combine different types of signals to get a better picture of what space really looks like; a common analogy is that of a garden: sight alone is not enough to fully experience it — one also needs to touch, smell, and hear it.

Reviewer: ```suggestion
```

> In the span of 9 years, only a few multi-messenger observations have been confirmed, but a huge amount of raw data, produced by the interferometers LIGO, VIRGO and KAGRA and neutrino detectors, are being analyzed. Neutrino and photonic observations have been useful to find blazars \cite{IceCube2018Multimessenger} for example.

Reviewer: ```suggestion
Since the detection of GW170817, there have not been more GW-light ulti-messenger events confirmed, despite the massive amount of raw data being analyzed by GW observatories like LIGO, VIRGO and KAGRA. In 2017, neutrino and photonic observations were successfully combined to identify blazars as high-energy sources \cite{IceCube2018Multimessenger}. The scientific community expects to drastically accelerate these discoveries in the coming nyears with next-generation observatories like Einstein Telescope, LISA, IceCube-Gen2.
```

> However there is still a long way to achieve efficient and daily observations and in the years to come the scientific community expects to intensify research and results thanks to the upcoming Einstein Telescope, LISA and IceCube-Gen2.

Reviewer: ```suggestion
```

> GAMMA contributes to this shared goal through its gamma-ray observations. More specifically, GAMMA fills the gap left by existing instruments, such as FERMI launched in 2008, which are decades old and have passed their operational lifetime; these telescopes don't fully cover the gamma ray frequency either, missing crucial information which our mission plans to acquire. The choice of GAMMA comes from this specific lack of instruments alongside the role that the gamma band plays in the multi-messenger era: gamma rays are the most energetic ones, thus we expect to observe them together with extreme events, that's exactly what happened with GW170817, and for this reason a follow up observation in this band rather than the others is very impactfull.

Reviewer: ```suggestion
\subsection{The GAMMA Mission}
GAMMA is designed to contribute directly to this shared goal through $\gamma$-ray observations. Specifically, GAMMA addresses the gap in high-energy transient monitoring. Legacy instruments, such as the Fermi Gamma-ray Space Telescope (launched in 2009), have long surpassed their original operational lifetimes. As these aging flagships eventually retire, there will be critical need for continuous, agile coverage of the $\gamma$-ray sky.

$\gamma$-rays represent the most energetic band of the electromagnetic spectrum and the immediate signature of extreme events, exactly as demonstrated by GW170817. Rapid follow-up observations in this specific band are therefore highly impactful for identifying electromagnetic counterparts to gravitational wave triggers.
```

> In conclusion, multimessenger astronomy opens new frontiers but requires more effort and instrumentation; a student-led mission can be as flexible as it needs while also keeping the right amount of rigor to be part of the complex scientific research.

Reviewer: ```suggestion
Ultimately, multi-messenger astronomy is opening new frontiers, but it requires constant evolution of instrumentation. A focused, student-led mission like GAMMA offers the flexibility and agility needed o provide targeted, high-value data, maintaining the rigorous scientific standards required to participate in modern astrophysical research.
```

%====================================================
\section{Multi-Messenger Astronomy}

> Multi-messenger astronomy (MMA) is the term that refers to a recently developed appraoch to studying the universe by combining different, independent sources of information. These sources consist of four fundamental 'messengers': electromagnetic radiation, cosmic rays, neutrinos, and gravitational waves.

Reviewer: ```suggestion
Multi-messenger astronomy (MMA) is the term that refers to a recently developed approach to studying the universe by combining different, independent sources of information. These sources consist of four fundamental 'messengers': electromagnetic radiation, cosmic rays, neutrinos, and gravitational waves.
```

> For centuries, the only 'messenger' used to observe cosmic events was light. Initially, limited to the optical range, observations eventually expanded across the entire electromagnetic spectrum, from radio waves to $\gamma$-rays. Because this spectrum is so vast, it must be divided into smaller bands observed with wavelength-specific telescopes. To gain a deeper understand of astrophysical sources, astronomers routinely combine data from these different bands (multi-band imaging).

Reviewer: ```suggestion
For centuries, the only 'messenger' used to observe cosmic events was light. Initially, limited to the optical range, observations eventually expanded across the entire electromagnetic spectrum, from radio waves to $\gamma$-rays. Because this spectrum is so vast, it must be divided into smaller bands observed with wavelength-specific telescopes. To gain a deeper understanding of astrophysical sources, astronomers routinely combine data from these different bands (multi-band imaging).
```

> Modern astrophysics has increasingle embraced the full multi-messenger approach moving beyond just photons \cite{2020_astro_decadal}. The universe provides critical information through other probes that carry unique signatures of extreme astrophysical phenomena. These include

Reviewer: ```suggestion
Modern astrophysics has increasingly embraced the full multi-messenger approach moving beyond just photons \cite{2020_astro_decadal}. The universe provides critical information through other probes that carry unique signatures of extreme astrophysical phenomena. These include
```

\begin{itemize}
    \item cosmic rays: charged particles travelling at relativistic speeds, originating from our sun and deep-space events \cite{missing}
    \item neutrinos: nearly massless particles that traverse the universe with minimal interaction, offering pristine and early insights into sources like supernovae, active galactic nuclei (AGN) and black holes \cite{missing}

>     \item gravitational waves (GWs): ripples in the fabric of space-time generated by the acceleration of massive, super-dense objects (black holes and neutron stars), detectable by highly sensitive interferometers \cite{missing}

Reviewer: add the missing references

\end{itemize}

> A paradigm shift in this field occurred on 17 August 2017 (GW170817), when a gravitational wave detection of a neutron star merger was directly followed by the observation of a gamma-ray burst (GRB) \cite{Goldstein2017}. From that moment on, the scientific community's goal has been to synthesize these different types of signals to construct a comprehensive picture of the cosmos. Relying on photons alone is no longer sufficient, observing extreme energy conditions requires the combined "senses" of all available messengers to get the full perspective.

Reviewer: for continuity across the document

```suggestion
A paradigm shift in this field occurred on 17 August 2017 (GW170817), when a gravitational wave detection of a binary neutron star merger was directly followed by the observation of a gamma-ray burst (GRB) \cite{Goldstein2017}. From that moment on, the scientific community's goal has been to synthesize these different types of signals to construct a comprehensive picture of the cosmos. Relying on photons alone is no longer sufficient, observing extreme energy conditions requires the combined 'senses' of all available messengers to get the full perspective.
```

> Since the detection of GW170817, there have not been more GW-light ulti-messenger events confirmed, despite the massive amount of raw data being analyzed by GW observatories like LIGO, VIRGO and KAGRA. In 2017, neutrino and photonic observations were successfully combined to identify blazars as high-energy sources \cite{IceCube2018Multimessenger}. The scientific community expects to drastically accelerate these discoveries in the coming nyears with next-generation observatories like Einstein Telescope, LISA, IceCube-Gen2.

Reviewer: - add citation of number of detected events through LIGE-VIRGO-KAGRA O1-O4 run
- add citation for the estimated detections with new generation observatories. here's an example for Einstein Telescope _[Colombo et al. 2025](https://arxiv.org/abs/2503.00116)_

```suggestion
Since the detection of GW170817, there have not been more GW-light ulti-messenger events confirmed, despite the massive amount of raw data being analyzed by GW observatories like LIGO, VIRGO and KAGRA \cite{missing}. In 2017, neutrino and photonic observations were successfully combined to identify blazars as high-energy sources \cite{IceCube2018Multimessenger}. The scientific community expects to drastically accelerate these discoveries in the coming years with next-generation observatories like Einstein Telescope, LISA, IceCube-Gen2. \cite{missing}
```

\subsection{The GAMMA Mission}
GAMMA is designed to contribute directly to this shared goal through $\gamma$-ray observations. Specifically, GAMMA addresses the gap in high-energy transient monitoring. Legacy instruments, such as the Fermi Gamma-ray Space Telescope (launched in 2009), have long surpassed their original operational lifetimes. As these aging flagships eventually retire, there will be critical need for continuous, agile coverage of the $\gamma$-ray sky.

> $\gamma$-rays represent the most energetic band of the electromagnetic spectrum and the immediate signature of extreme events, exactly as demonstrated by GW170817. Rapid follow-up observations in this specific band are therefore highly impactful for identifying electromagnetic counterparts to gravitational wave triggers.

Reviewer: add citation about the importance of GRBs as early-alerts/localisations for rapid follow-ups to GW events, maybe there is some information in this half-related paper: _[Mastrogiovanni et al. 2022](https://onlinelibrary.wiley.com/doi/10.1002/andp.202200180)_

hint: GW actually don't provide good enough localisation for ground-based telescopes (narrow FoV), GRBs telescope normally have wider FoV and provide better localisation

```suggestion
$\gamma$-rays represent the most energetic band of the electromagnetic spectrum and the immediate signature of extreme events, exactly as demonstrated by GW170817. Rapid follow-up observations in this specific band are therefore highly impactful for identifying electromagnetic counterparts to gravitational wave triggers. \cite{missing}
```

> Ultimately, multi-messenger astronomy is opening new frontiers, but it requires constant evolution of instrumentation. A focused, student-led mission like GAMMA offers the flexibility and agility needed o provide targeted, high-value data, maintaining the rigorous scientific standards required to participate in modern astrophysical research.

Reviewer: ```suggestion
Ultimately, multi-messenger astronomy is opening new frontiers, but it requires constant evolution of instrumentation. A focused, student-led mission like GAMMA offers the flexibility and agility needed to provide targeted, high-value data, maintaining the rigorous scientific standards required to participate in modern astrophysical research.
```

why/how does CAPIBARA-GAMMA provide flexibility and agility @alesab-08 ?
